engine/src/main/java/org/pentaho/di/trans/steps/simplemapping/SimpleMappingMeta.java

```diff
@@ -76,14 +78,18 @@
  * @author Matt
  *
  */
-
+@InjectionSupported( localizationPrefix = "SimpleMappingMeta.Injection.", groups = { "PARAMETERS", "INPUTS" } )
 public class SimpleMappingMeta extends StepWithMappingMeta implements StepMetaInterface, HasRepositoryInterface,
   ISubTransAwareMeta {
 
   private static Class<?> PKG = SimpleMappingMeta.class; // for i18n purposes, needed by Translator2!!
 
+  @InjectionDeep
   private MappingIODefinition inputMapping;
+
   private MappingIODefinition outputMapping;
```

Comment:
Is there a reason this was left out of the MDI?

Reply:
Yes. According to https://docs.google.com/spreadsheets/d/1sdnMKXcEAOe5A7-76ex1ll3Gbp51ZVGTE7lsU9DRUHs/edit#gid=375223105 only the input part is considered.

Reply:
Sounds good :)